common/KeeNotification.ts

```diff
@@ -43,6 +44,22 @@ class KeeNotification {
         return container;
     }
 
+    renderCloseButton (container: HTMLDivElement) {
+        const button = document.createElement("span");
+        
+        button.classList.add("close-button", "glyphicon", "glyphicon-remove");
+        // TODO: should we add hint to this obvious action button? 
+        // in this case we need i18n for this hint
+        // button.setAttribute("title", "Hide notification");
```

Comment:
Yes, we should do that. We could think about whether there is a more suitable text string in future but for today, let's use the existing "close" string:

`button.setAttribute("title", $STR("close"));`